packages/blade/src/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import React from 'react';
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Button, Text } from '..';
+import type { RadioProps } from './Radio';
+import { Radio as RadioComponent } from './Radio';
+import Box from '~components/Box';
+
+export default {
+  title: 'Components/Radio/Radio',
+  component: RadioComponent,
+  args: {
```

Comment:
I thought we don't need to specify `undefined` props below 🤔

Reply:
Yeah wouldn't matter if we don't, just the sorting will be random.